.github/workflows/build.yaml

```diff
@@ -62,5 +65,14 @@ jobs:
         cargo clippy --version
         cargo fmt --version
 
-    - name: CI
-      run: ./bin/ci
+    - name: Build
+      run: cargo build --all
+
+    - name: Test
+      run: cargo test --all
+
+    - name: Clippy
+      run: cargo clippy --all
```

Comment:
I've 
It can be fun to `+nightly` and get all the new lints.
```
clippy:
	cargo +nightly clippy --all-targets
```
I also have this in my `src/lib`:
```
#![warn(
    clippy::nursery,
    //clippy::restriction,
    //clippy::pedantic,
    //clippy::cargo,
)]
```
I only use the nursery lints, but it can be fun to see the other lints so I turn them on and off from time to time. 
All of this said, it's been a while since I've poked around in clippy land. There might be some new, cool ways of doing the same thing.